### PR TITLE
MTM-52839 New design Application solution & Enablement

### PR DESCRIPTION
**Pull request by MWindel**
Re-structured the different parts in the section "Application Solution and Enablement"

### CODE REVIEW

File: content/web.md [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
-section: 
+section:
   - app_development
 bundle: web
 weight: 50
----
\ No newline at end of file
+---
+
+Find out more on how to develop web applications for your Cumulocity IoT platform, connect and communicate with our API as well as integrate UI components into your custom application.
```
Comment (BeateRixen) [inline content/web.md:12]:
```suggestion
Discover how to develop web applications with the Web SDK and integrate default or branded UI components into your custom application.
```

File: content/microservice-sdk.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Microservice SDK guide
+title: Developing microservices with Microservice SDK
```
Comment (BeateRixen) [inline content/microservice-sdk.md:2]:
```suggestion
title: Developing and deploying microservices
```

File: content/microservice-sdk.md [hunk 3]
```diff
@@ -8,3 +8,5 @@ section:
   - app_development
 weight: 60
 ---
+
+Learn how to extend your custom application using microservices.
```
Comment (BeateRixen) [inline content/microservice-sdk.md:12]:
```suggestion
Learn how to extend your platform with customer-specific functionality by developing and deploying microservices.
```

Comment (BeateRixen) [reply content/microservice-sdk.md:12]: We must try to get the descriptions on the tiles more or less the same size (ideally 3 rows).

File: content/oee.md [hunk 4]
```diff
@@ -1,10 +1,12 @@
...
 type: root
 layout: redirect
 bundle: oee
-section: 
+section:
   - app_development
 weight: 80
 ---
+
+Use the Cumulocity IoT OEE application to measure the efficiency and effectiveness of your production processes, to create calculation configurations and to receive and store data through our API.
```
Comment (BeateRixen) [inline content/oee.md:12]:
```suggestion
Use the OEE application to measure the efficiency and effectiveness of your production processes, to create calculation configurations and to receive and store data through our API.
```

File: content/application-and-microservice-management/extensions-ecosystem.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+---
+title: Ecosystem
```
[the remark this answers is not in the document]
Comment (MWindel) [reply content/application-and-microservice-management/extensions-ecosystem.md:2]:
```suggestion
title: Extensions
```

File: content/application-and-microservice-management/default-subscriptions-ecosystem.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
+---
+title: Default subscriptions
```
[the remark this answers is not in the document]
Comment (MWindel) [reply content/application-and-microservice-management/default-subscriptions-ecosystem.md:2]: I copied this structure from other single .md-files. They are also in the layout "bundle".

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**